dev/snippets/config/templates/stateful_widget.tmpl

```diff
@@ -10,12 +10,13 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return new MaterialApp(
+    return MaterialApp(
       title: 'Flutter Code Sample for {{id}}',
-      theme: new ThemeData(
+      theme: ThemeData(
```

Comment:
There's no need to specify the default theme here. I assume that we include it in `flutter create` because we're trying remind developers that it's an option and that changing the primary color is easy. In this case, the goal is to highlight the snippet, so it doesn't seem necessary.